goosebit/__init__.py

```diff
@@ -28,7 +24,24 @@ async def lifespan(_: FastAPI):
     await db.close()
 
 
-app = FastAPI(lifespan=lifespan)
+app = FastAPI(
+    title="gooseBit",
+    summary="A simplistic, opinionated remote update server implementing hawkBit™'s DDI API.",
+    version=importlib.metadata.version("goosebit"),
+    lifespan=lifespan,
+    license_info={
+        "name": "Apache 2.0",
+        "identifier": "Apache-2.0",
+    },
+    swagger_ui_parameters={"operationsSorter": "alpha"},
```

Comment:
Currently, we have Swagger UI (/docs) and ReDoc (/redoc). I think ReDoc looks quite nice but, doesn't seem to show the schemas. Anyway, I think we should decide on one and disable the other.

Reply:
My preference is swagger, mainly because of the schemas as mentioned.  I will try to put forward another PR soon that will customize the swagger UI a bit further, including the favicon, and I can remove redoc there.